Form-Controls/index.html

```diff
@@ -9,19 +9,70 @@
   </head>
   <body>
     <header>
-      <h1>Product Pick</h1>
+      <h1>The T-shirt Store</h1>
     </header>
     <main>
-      <form>
-        <!-- write your html here-->
-        <!-- 
-          try writing out the requirements first as comments
-        this will also help you fill in your PR message later-->
+      <form id="tshirt-form" action="" method="post">
+
+        <!-- 1. customer's name? validate it. But what is a valid name? I must decide something.
```

Comment:
These comments can be removed as they don't provide any additional information to a future developer. The developer is able to understand your from you code that you are validating form input fields.